### ISSUE TITLE
JSON config : Add possibility to use "sendTo" for checkboxes

### ISSUE THREAD

**Issue by DutchmanNL** [ioBroker/ioBroker.admin#1136]
Is your feature request related to a problem? Please describe.
Several adapter use checkboxes in there admin config which allows to enabled specific elements only available at that system or in relation to conditions
(like SEQ,CoronaVirusStatistis), 
I would like to rebuild these adapter to use admin 5 with JSON config.

Describe the solution you'd like
Currently I use javascript to retrieve objects which build the checkboxes, so I need a Dynamic way to remove/add values to checkboxes instead of the current static one (preferable by just sending the data as JSON by messaging equal to selectSendto

Describe alternatives you've considered
I don't see an alternative as the current json config does only allow static checkboxes, would like to make them dynamic

example to use: CoronaVirus statistics:

<img width="1476" alt="Schermafbeelding 2021-08-25 om 11 33 53" src="https://user-images.githubusercontent.com/7318445/130766682-bcdd9d37-54b9-493c-9a97-bf025ffb4409.png">

**Comment by GermanBluefox** [comment]
How it is stored?
```
"native": {
    "Algeria": true,
    "Bahrain": false,
   ....
}
```
Or as array?

**Comment by GermanBluefox** [comment]
@DutchmanNL If you will not provide me a use-case, I will not be able to implement it. Kind of select with multiple choice?

**Comment by GermanBluefox** [comment]
What about such a selection?
![image](https://user-images.githubusercontent.com/4582016/174862096-f0d9828e-9af3-4336-869d-2874b0ae5476.png)

Already implemented in 6.0.8

**Comment by GermanBluefox** [comment]
![image](https://user-images.githubusercontent.com/4582016/174866241-236adea4-20e9-498e-8c78-a1cd68a2033c.png)

**Comment by DutchmanNL** [comment]
Already implemented in 6.0.8
